fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java

```diff
@@ -305,9 +305,7 @@ public static DataProperty analyzeDataProperty(Map<String, String> properties, f
             // check remote storage cool down timestamp
             if (storagePolicy.getCooldownTimestampMs() != -1) {
                 if (storagePolicy.getCooldownTimestampMs() <= currentTimeMs) {
-                    throw new AnalysisException(
```

Comment:
Do not modify it like this.
1. Should check the property when create or alter the storage policy if the cooldown time is less than now.
2. Throw exception when create table if the cooldown time is less than now.